tools/waveica/waveica_wrapper.R

```diff
@@ -149,7 +217,12 @@ exclude_group <- function(data, group) {
 
 
 # Store output of WaveICA in a tsv file
```

Comment:
This comment can be updated or removed

Reply:
resolved in [9bee08a](https://github.com/RECETOX/galaxytools/pull/304/commits/9bee08ace9062a191a0b6f43344ff4f078517345)